solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -48,9 +48,8 @@ export class DataSetService implements DataSetPlugin {
   public addDataSetExternalEndpoint(
     request: DataSetExternalEndpointRequest
   ): Promise<DataSetAddExternalEndpointResponse> {
-    return this._workbenchDataSetService.addDataSetExternalEndpointForUser({
+    return this._workbenchDataSetService.addDataSetExternalEndpointForGroup({
```

Comment:
do they only accept a group here? never a user? If they expose `identityType`, you should check that and call the appropriate method based on type.

Reply:
@kpark277 SWB is only using `addDataSetExternalEndpointForGroup()` right?